packages/express/lib/circuit-breaker.js

```diff
@@ -0,0 +1,62 @@
+const { performance } = require('perf_hooks');
+
+// TODO: figure out good values
+const upperThreshold = 0.75;
+const lowerThreshold = 0.6;
+
+function createCircuitBreaker(app) {
+  let elu = performance.eventLoopUtilization();
+  app.locals.breaker = 'closed';
+  app.locals.noSSR = false;
+
+  let last = performance.now();
+
+  return (req, res, next) => {
+    elu = performance.eventLoopUtilization();
```

Comment:
Right now we're calculating the ELU per each request, I'm not sure, if we should try to implement some sort of sliding-window over some configurable amount of time or whether we should take it each request.
Also: I'm not entirely sure, if we maybe should pass in an older ELU value into the `eventLoopUtilization()` method, to calculate a delta between two values?